docs/best-practices/continuous-integration.md

```diff
@@ -446,3 +446,57 @@ The extension can be configured to install the latest version of _fastlane_ or a
 
 ## More Information
 For more documentation, see the [Apple App Store](https://marketplace.visualstudio.com/items?itemName=ms-vsclient.app-store) in the Visual Studio Team Server marketplace.
+
+# Nevercode Integration for iOS
+
+[Nevercode](www.nevercode.io) expects nothing more than a [`Fastfile`](https://docs.fastlane.tools/#fastlane) in your repository. 
+
+## Specify app repository
+
+1. Click **Add a new app** on Nevercode dashboard.
+
```

Comment:
This new line breaks the list in markdown

Reply:
Hi @KrauseFx ! Thanks for the feedback! Super excited to have fastlane with Nevercode. Keep up the good work! 
This improvement done!